input/pagecontent/layers.md

```diff
@@ -61,8 +62,38 @@ Example view runners may include:
 
 Finally, users must be able to easily leverage the above views with the analytic
 tools of their choice. This spec purposefully does not define what these are,
-but common use cases may be SQL queries by consuming applications or integration
-with business intelligence or data science tools.
+but common use cases may be SQL queries by consuming applications, dataframe-based data science tools in Python or R, or integration
+with business intelligence tools.
+
+
+
+### Archimate layered viewpoint
```

Comment:
I wonder if this is overspecying the data layer here, and we should consider moving data layer details like this to a "guidance" page, rather than as part of the layer descriptions. 

My reasoning is that we wanted to leave a great deal of flexibility to data layer implementations, as there could be new ways to efficiently represent FHIR data for analytic workloads that does not fit any of the descriptions here. But, if folks have a particular tech stack in mind, we can share guidance for an NDJSON-based runner vs. a Postgres runner vs a Spark runner, and so on.

So I think my inclination would be to leave these details out of this page, but move the relevant content into guidance?

Reply:
@rbrush thanks for the feedback. I am easy either way, that is, whether the description of the layer stays here or moves into a guidance section.

My main motivation was to make it explicit. I often find I need these details to explain to stakeholders what the specification is trying to achieve and how it plays out it practice.

So please let me know what the consensus is and I will make changes accordingly.

Reply:
@dkapitan First of all, thanks very much for your work on this, and I really appreciate you taking the time to think about this and get involved!

I don't think there is any problem with having a detailed description of the layers and architecture here, but I think it should be described in a way that is completely implementation-agnostic.

View runner implementations, and specific implementations within the data layer and the analytics layer are outside the scope of this specification - we don't explicitly support or preclude anything in this space. We had some of this stuff in draft versions of the spec as we were throwing ideas around, but I think as we progress this towards a draft we need to be really clear about this.

I think that examples are extremely useful, but I think they should be separate from the main description and should be labelled as examples. I think that examples of the realisation of the architecture would be very helpful for readers.

Reply:
Maybe we could add a section called "Examples", and extract all the example-related content here.

Then we could reword the description of the layers to talk exclusively in terms of components and their roles at a logical level.

That's just my opinion, happy to hear from others.

Reply:
Separating these out into an "examples" section, or under different guidance pages as I suggested earlier makes sense to me as well. 

Maybe for now we could make the "Archimate layered viewpoint" as an "Example" section would make sense, and we can move it later if needed? 

The high-level diagram and content updates look great, so I'd love to merge this if we are a bit more clear on these details are examples (and other implementations may take a very different approach).